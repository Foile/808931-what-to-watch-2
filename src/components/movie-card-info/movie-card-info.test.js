import React from 'react';
import renderer from "react-test-renderer";
import MovieCardInfo from './movie-card-info';

it(`MovieCardInfo correctly renders after relaunch`, () => {
  const movie = {id: 1, name: `Macbeth`, previewImage: ``, videoLink: ``};
  const tree = renderer
  .create(<MovieCardInfo
    films = {movie}
    review = {true}
    auth = {{}} />)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
