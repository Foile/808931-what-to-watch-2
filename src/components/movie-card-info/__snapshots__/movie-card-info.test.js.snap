// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCardInfo correctly renders after relaunch 1`] = `null`;
